Framework/Muon/src/EstimateMuonAsymmetryFromCounts.cpp

```diff
@@ -68,7 +68,8 @@ void EstimateMuonAsymmetryFromCounts::init() {
 
   declareProperty(
       make_unique<API::WorkspaceProperty<API::ITableWorkspace>>(
-          "NormalizationTable", "", Direction::InOut),
+          "NormalizationTable", "", Direction::InOut,
+          API::PropertyMode::Optional),
```

Comment:
long term, this is probably a good idea. However, it is presently the only way to record the normalization. Since we need to know what the value is later on it is worth recording. As far as I can see we have two choices:

1. Need a table to be provided.
2. Regardless to if a table is provided or not we add the normalization to the sample logs of the output ws.